### ISSUE TITLE
Still Files Compilation

### ISSUE THREAD

**Issue by awinia-github** [Semi-ATE/Semi-ATE#259]
Blocked by #256 

The test program provides currently an option to compile stil files, this is still unimplemented.

A.C:
* trigger compile option shall start the stil compiler
* stil plugin shall be enabled (automaitcally) to visualize the compilation prozess  (nice to have)

__note__: see 'flowitem.py' -> ('compile_pattern_files' function)
